pkg/flowkit/gateway/gateway.go

```diff
@@ -38,5 +38,6 @@ type Gateway interface {
 	GetBlockByID(flow.Identifier) (*flow.Block, error)
 	GetEvents(string, uint64, uint64) ([]client.BlockEvents, error)
 	GetCollection(flow.Identifier) (*flow.Collection, error)
+	GetLatestProtocolStateSnapshot() ([]byte, error)
```

Comment:
I believe you should add this method to the emulator too https://github.com/onflow/flow-cli/blob/master/pkg/flowkit/gateway/emulator.go you can just add `return nil, ftm.Errorf("implement me")`

Reply:
I've done that see file right above this one ^

Reply:
Sorry you are right.